pyvisa/util.py

```diff
@@ -183,37 +235,16 @@ def __new__(
         return obj
 
     @property
-    def arch(self) -> Tuple[int, ...]:
+    def arch(self) -> List[ArchitectureType]:
         """Architecture of the library."""
         if self._arch is None:
             try:
-                self._arch = get_arch(self.path)
+                self._arch = get_arch(Path(self.path))
             except Exception:
-                self._arch = tuple()
+                self._arch = []
 
         return self._arch
 
-    @property
-    def is_32bit(self) -> Union[bool, Literal["n/a"]]:
-        """Is the library 32 bits."""
-        if not self.arch:
-            return "n/a"
-        return 32 in self.arch
-
-    @property
-    def is_64bit(self) -> Union[bool, Literal["n/a"]]:
-        """Is the library 64 bits."""
-        if not self.arch:
-            return "n/a"
-        return 64 in self.arch
-
-    @property
-    def bitness(self) -> str:
-        """Bitness of the library."""
-        if not self.arch:
-            return "n/a"
-        return ", ".join(str(a) for a in self.arch)
-
```

Comment:
This is technically backward incompatible... But it is far from the core so we can ride with it.

Can you please add an entry to the changelog mentioning that those were removed.